localisation/000_wc_military_localization.csv

```diff
@@ -1,17 +1,13 @@
 #CODE;ENGLISH;FRENCH;GERMAN;;SPANISH;;;;;;;;;x
 #Military Technologies;;;;;;;;;;;;;;x
-TECH_INFANTRY;Conventional Warfare;;Konventionelle Kriegsf�hrung;;;;;;;;;;;x
-TECH_CAVALRY;Specialized Warfare;;Irregul�re Kriegsf�hrung;;;;;;;;;;;x
-TECH_SKIRMISH;Ley Attunement;;Ley Einstimmung;;;;;;;;;;;x
-TECH_MELEE;Tactics Development;;Taktikentwicklung;;;;;;;;;;;x
+TECH_INFANTRY;Frontline Troops;;;;;;;;;;;;;x
+TECH_CAVALRY;Charging Troops;;;;;;;;;;;;;x
 TECH_SIEGE_EQUIPMENT;Military Engineering;;Wehrtechnik;;;;;;;;;;;x
 TECH_RECRUITMENT;Martial Logistics;;Kriegslogistik;;;;;;;;;;;x
-TECH_INFANTRY_DESC;Better equipment and discipline for our conventional troops;;Bessere Ausr�stung und Disziplin f�r unsere konventionellen Truppen;;;;;;;;;;;x
-TECH_CAVALRY_DESC;Better equipment and discipline for our more unconventional troops;;Bessere Ausr�stung und Disziplin f�r unsere Sondereinheiten;;;;;;;;;;;x
-TECH_SKIRMISH_DESC;Better training and discipline for our magically inclined troops;;Bessere Ausbildung und Disziplin f�r unsere magisch veranlagten Truppen;;;;;;;;;;;x
-TECH_MELEE_DESC;More developed and modern warfare tactics;;Entwickelte und moderne Kriegstaktiken;;;;;;;;;;;x
-TECH_SIEGE_EQUIPMENT_DESC;More advanced siege techniques and equipment for our machine troops;;Fortschrittlichere Belagerungstechniken und -Ausr�stung f�r unsere mechanischen Truppen;;;;;;;;;;;x
-TECH_RECRUITMENT_DESC;More developed logistic systems and recruitment methods;;Weiterentwickelte Logistiksysteme und Rekrutierungsmethoden;;;;;;;;;;;x
+TECH_INFANTRY_DESC;Better Equipment and Discipline for our Frontline Troops;;;;;;;;;;;;;x
```

Comment:
I may not particularly say that Casters and Archers are "frontline". Maybe Primary Troops, Baseline Troops, or Principal Troops?